database/seedData.js

```diff
@@ -1,14 +1,8 @@
 const mongoose = require('mongoose');
+const faker = require('faker');
 
-mongoose.connect('mongodb://localhost/n3rdstorm', (err, db) => {
-  if (err) {
-    console.error(err);
-  }
-  db.dropDatabase();
-  db.close();
-});
-
-mongoose.connect('mongodb://localhost/n3rdstorm');
+mongoose.connect('mongodb://localhost/n3rdstorm', () => mongoose.co);
```

Comment:
What does mongoose.co do in the callback?

Reply:
Oops, I mistakenly left that in there. Removed now.